### ISSUE TITLE
[Search bar in the middle][Accessibility] VoiceOver focus moves after dismissing keyboard and text input

### ISSUE THREAD

**Issue by natastro** [mozilla-mobile/firefox-ios#28109]
## Prerequisites:
* Turn on VoiceOver
* Turn on 'Homepage Search Bar' located in the Feature Flag menu
* Found in Firefox Beta 141.0 on iOS

## STR:
1. Turn on VoiceOver
1. Open Firefox Beta app
1. Navigate to the 'Search or enter address button'. Double tap to activate
1. A text input with a keyboard should appear on screen with focus starting in the text input. Begin to type a search query starting with the first letter
1. After you've typed the first letter, move VoiceOver focus to the 'Cancel' button and double tap to activate
1. Observe VoiceOver focus behavior

## Actual: 
After tapping the 'Cancel' button, VoiceOver focus does not return to the 'Search or enter address button'

## Expected:
Once something is dismissed on screen, VoiceOver focus should return to the control that opened the keyboard + text input

## Device & build:
* Device: iPhone 15, iOS 18.5
* Build version: Firefox Beta 141.0
* First seen version: First time I've seen this issue

https://github.com/user-attachments/assets/70fd1018-f702-4975-8f5e-228424aec448

First seen version: First time I've seen this issue

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-12893)

**Comment by thatswinnie** [comment]
Closing as search bar in the middle was removed.
